saltstack/optional/pillar/top.sls

```diff
@@ -1,3 +1,4 @@
 optional:
   '*':
-    - dummy
\ No newline at end of file
+    - dummy
```

Comment:
If you add oracle-java then dummy should be removed.

Reply:
Corrected, thanks.